match2/wikis/rainbow_six/match_summary.lua

```diff
@@ -0,0 +1,579 @@
+		team1Score:setSecondRoundScore(oppositeSide, team1Halfs[oppositeSide], true)
+
+		-- Overtime for both, if applicable
+		local firstSideOvertime = firstSides.ot
+		local oppositeSideOvertime = CustomMatchSummary._getOppositeSide(firstSideOvertime)
+
+		if not Logic.isEmpty(firstSideOvertime) then
+			team1Score:setFirstOvertimeRoundScore(firstSideOvertime, team1Halfs['ot'..firstSide], true)
+			team1Score:setSecondOvertimeRoundScore(oppositeSideOvertime, team1Halfs['ot'..oppositeSide], true)
+
+			team2Score:setFirstOvertimeRoundScore(oppositeSideOvertime, team2Halfs['ot'..oppositeSide], false)
+			team2Score:setSecondOvertimeRoundScore(firstSideOvertime, team2Halfs['ot'..firstSide], false)
+		else
+			team1Score:addEmptyOvertime()
+			team2Score:addEmptyOvertime()
+		end
+
+		-- Regular Time for Team 2
+		team2Score:setFirstRoundScore(oppositeSide, team2Halfs[oppositeSide], false)
+		team2Score:setSecondRoundScore(firstSide, team2Halfs[firstSide], false)
+	end
+
+	-- Score Team 2
+	team2Score:setMapScore(game.scores[2])
+
+	-- Operator bans
+	local operatorBans = {team1 = Json.parse(extradata.t1bans) or {}, team2 = Json.parse(extradata.t2bans) or {}}
+	local team1OperatorBans = OperatorBans():setLeft()
+	local team2OperatorBans = OperatorBans():setRight()
+
+	for _, operator in ipairs(operatorBans.team1) do
+		team1OperatorBans:add(operator)
+	end
+	for _, operator in ipairs(operatorBans.team2) do
+		team2OperatorBans:add(operator)
+	end
+
+	-- Add everything to view
+	if team1OperatorBans ~= nil then
+		row:addElement(team1OperatorBans:create())
+	end
+	row:addElement(CustomMatchSummary._createCheckMark(game.winner == 1))
+	row:addElement(team1Score:create())
+
+	local centerNode = mw.html.create('div')
+	centerNode  :addClass('brkts-popup-spaced')
+				:addClass('bracket-popup-body-match-map')
+				:wikitext('[[' .. game.map .. ']]')
+				:css('text-align', 'center')
+				:css('padding','5px 2px')
+				:css('flex-grow','1')
+
+	if game.resultType == 'np' then
+		centerNode:addClass('brkts-popup-spaced-map-skip')
+	end
+
+	row:addElement(centerNode)
+	row:addElement(team2Score:create())
+	row:addElement(CustomMatchSummary._createCheckMark(game.winner == 2))
+	if team2OperatorBans ~= nil then
+		row:addElement(team2OperatorBans:create())
+	end
+
+	-- Add Comment
+	if not Logic.isEmpty(game.comment) then
+		row:addElement(MatchSummary.Break():create())
+		local comment = mw.html.create('div')
+		comment :wikitext(game.comment)
+				:css('margin', 'auto')
+		row:addElement(comment)
+	end
+
+	row:addClass('brkts-popup-body-game'):css('font-size', '85%')
+
+	-- Winner/Loser backgrouns
+	if game.winner == 1 then
+		row:addClass('bracket-popup-body-gradient-left')
+	elseif game.winner == 2 then
+		row:addClass('bracket-popup-body-gradient-right')
+	elseif game.winner == 'draw' then
+		row:addClass('bracket-popup-body-gradient-draw')
+	else
+		row:addClass('bracket-popup-body-gradient-default')
```

Comment:
those classes are part of https://liquipedia.net/commons/MediaWiki:Common.css/Bracket.css
might need to be moved to https://liquipedia.net/commons/MediaWiki:Common.css/Brackets.css (and renamed to fit the naming pattern -> brkts instead of bracket)

before moving them please check if classes that do the same already exist in the `Brackets.css` file

(also applies for some of the above used classes)

Reply:
Moved to brkts